.env.tmpl

```diff
@@ -12,10 +12,10 @@ CKAN___BEAKER__SESSION__SECRET=
 CKAN___APP_INSTANCE_UUID=
 
 # Database config
-CKAN__DATASTORE_WRITE_URL=postgres://ckan_default:pass@db/datastore_default
-CKAN__DATASTORE_READ_URL=postgres://datastore_default:pass@db/datastore_default
+CKAN_DATASTORE_WRITE_URL=postgres://ckan_default:pass@db/datastore_default
+CKAN_DATASTORE_READ_URL=postgres://datastore_default:pass@db/datastore_default
 
-CKAN__REDIS__URL=redis://redis:6379/0
+CKAN_REDIS_URL=redis://redis:6379/0
```

Comment:
Why are we changing the `__` to `_`?

Reply:
I think this is what the ckan code expects; not sure why we had the `__`